src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+import { ErrorTypes } from '../../enum/ErrorTypes';
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+import { getSortedPeople } from '../../utils/getSortedPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
![image](https://github.com/mate-academy/react_people-table-advanced/assets/96409563/f01303a8-a6a6-4941-9678-43b2898a94e1)